circle.yml

```diff
@@ -0,0 +1,44 @@
+machine:
+  python:
+    version: 2.7.10
+
+dependencies:
+  cache_directories:
+    - ../examples/
+
+  pre:
+    - sudo pip install --upgrade pip
+    - sudo pip install traitlets pexpect pickleshare simplegeneric
+    - cd .. && wget https://pypi.python.org/packages/source/i/ipython/ipython-4.1.1.tar.gz && tar -xzf ipython-4.1.1.tar.gz && cd ipython-4.1.1 && sudo python setup.py install
+    - sudo pip install sphinx==1.2.3 pygraphviz matplotlib Pillow neo scipy numpy
+    - cd .. && git clone https://github.com/sphinx-gallery/sphinx-gallery.git && cd sphinx-gallery && sudo pip install -r requirements.txt && sudo python setup.py develop
+    - sudo pip install sphinx_bootstrap_theme
+    - sudo pip install numpy --upgrade
```

Comment:
Oof, pip installing `numpy` and `scipy` (which it seems like you'll need to do but I don't see it?) is rough and will be not optimized very much. Feel free to comment here to see if maybe they'll let us use Anaconda some day:

https://discuss.circleci.com/t/disable-autodetection-of-project-or-application-of-python-venv/235/2